Lib/python/std_unordered_multimap.i

```diff
@@ -20,10 +20,8 @@
 	int res = SWIG_ERROR;
 	if (PyDict_Check(obj)) {
 	  SwigVar_PyObject items = PyObject_CallMethod(obj,(char *)"items",NULL);
-%#if PY_VERSION_HEX >= 0x03000000
           /* In Python 3.x the ".items()" method returns a dict_items object */
           items = PySequence_Fast(items, ".items() didn't return a sequence!");
-%#endif
```

Comment:
The code which was inside the removed conditional has an extra indent, which no longer makes sense.